Wdvh/absolutecard/absolutecard.py

```diff
@@ -0,0 +1,456 @@
+from pathlib import Path
+from re import findall
+
+from modules.CardType import CardType
+from modules.Debug import log
+
+class StandardTitleCard(CardType):
+    """
+    This class describes a type of CardType that produces the 'generic' title
+    cards based on Reddit user /u/UniversalPolymath. This card supports 
+    customization of every aspect of the card, but does not use any arbitrary
+    data.
+    """
+
+    """Directory where all reference files used by this card are stored"""
+    REF_DIRECTORY = Path(__file__).parent / 'ref'
```

Comment:
```suggestion
    REF_DIRECTORY = Path(__file__).parent.parent / 'ref'
```
Need to change this to work as a GitHub-hosted CardType